app/src/main/java/org/p2p/wallet/sdk/facade/SolendMethodResultHandler.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.wallet.sdk.facade
+
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultError
+import org.p2p.wallet.sdk.facade.model.SolendMethodResultSuccess
+
+interface SolendMethodResultHandler {
+    fun handleResultSuccess(result: SolendMethodResultSuccess)
+    fun handleResultError(error: SolendMethodResultError)
```

Comment:
Don't really like the callbacks for catching the call's result.
It's absolutely different from our current way of making requests.

The developer should not see any difference between calling the simple REST request and SolendSDK call.
Let's try to create such experience

Reply:
will do